### ISSUE TITLE
大佬，我用案例工作流生成图转视频，无论调用i2v还是insight生成的图转视频都是花屏的

### ISSUE THREAD

**Issue by xlibfly** [eddyhhlure1Eddy/ode-ComfyUI-WanVideoWrapper#2]
请问是否是我的依赖有问题呢？

<img width="1059" height="1140" alt="Image" src="https://github.com/user-attachments/assets/5b30d75a-f844-42a1-89dd-a54215031d70" />

<img width="1198" height="82" alt="Image" src="https://github.com/user-attachments/assets/c957d797-e875-42b7-a1ae-81f3c9e7cc93" />

# Python环境: G:/AI-Tools/ComfyUI-aki-v1.6/python/python.exe
# 生成时间: 2025-09-30 11:38:08
# 共安装了 591 个包
#
# 包列表:
Brotli==1.1.0
Cython==3.1.2
DeepCache==0.1.1
Deprecated==1.2.18
Flask==3.1.1
GitPython==3.1.44
HyperPyYAML==1.2.2
Jinja2==3.1.6
LangSegment==0.2.0
Mako==1.3.10
Markdown==3.8.2
MarkupSafe==3.0.2
PyGithub==2.5.0
PyJWT==2.8.0
PyMatting==1.1.13
PyMuPDF==1.26.1
PyNaCl==1.5.0
PyOpenGL-accelerate==3.1.9
PyOpenGL==3.1.9
PyPDF2==3.0.1
PySocks==1.7.1
PyWavelets==1.8.0
PyYAML==6.0.2
Pygments==2.19.1
Rtree==1.3.0
SAM-2==1.0
SQLAlchemy==2.0.41
Send2Trash==1.8.3
Unidecode==1.4.0
Werkzeug==3.1.3
absl-py==2.1.0
accelerate==1.10.0
addict==2.4.0
aggdraw==1.3.19
aiofiles==24.1.0
aiohappyeyeballs==2.4.4
aiohttp==3.11.11
aiohttp_socks==0.10.1
aiosignal==1.3.2
aiosqlite==0.21.0
aisuite==0.1.11
albucore==0.0.24
albumentations==2.0.8
alembic==1.16.2
altair==5.5.0
annotated-types==0.7.0
anthropic==0.30.1
antlr4-python3-runtime==4.9.3
anyio==4.8.0
argcomplete==3.6.2
argostranslate==1.9.6
arxiv==2.2.0
asttokens==3.0.0
astunparse==1.6.3
attrdict==2.0.1
attrs==25.1.0
audioread==3.0.1
audiosegment==0.23.0
av==14.2.0
babel==2.17.0
banks==2.2.0
beautifulsoup4==4.13.1
bidict==0.23.1
bitsandbytes==0.46.0
black==25.1.0
blend_modes==2.2.0
blind-watermark==0.4.4
blinker==1.9.0
boto3==1.38.40
botocore==1.38.40
bs4==0.0.2
cachetools==5.5.1
came-pytorch==0.1.3
cerebras_cloud_sdk==1.46.0
certifi==2025.8.3
cffi==1.17.1
cfgv==3.4.0
chardet==5.2.0
charset-normalizer==3.4.1
click==8.1.8
clip-interrogator==0.6.0
cmake==3.31.4
cn2an==0.5.23
cohere==5.17.0
color-matcher==0.5.0
colorama==0.4.6
coloredlogs==15.0.1
colorlog==6.9.0
colour-science==0.4.6
comfyui-embedded-docs==0.2.6
comfyui_frontend_package==1.26.13
comfyui_workflow_templates==0.1.91
compel==2.1.1
conformer==0.3.2
contourpy==1.3.1
cpm-kernels==1.0.11
cryptography==44.0.0
cssselect2==0.7.0
cstr==0.1.0
csvw==3.5.1
ctranslate2==4.6.0
cupy-cuda12x==13.6.0
cycler==0.12.1
dataclasses-json==0.6.7
datamodel-code-generator==0.31.1
ddt==1.7.2
decorator==4.4.2
decord==0.6.0
deep-translator==1.11.4
deepdiff==8.2.0
deepface==0.0.93
defusedxml==0.7.1
diffusers==0.35.1
dill==0.3.9
dirtyjson==1.0.8
diskcache==5.6.3
distlib==0.3.9
distro==1.9.0
dlib==19.24.6
dlinfo==2.0.0
docopt==0.6.2
docstring_parser==0.16
docutils==0.21.2
docx2txt==0.9
dpcpp-cpp-rt==2025.1.1
dynamicprompts==0.30.2
easydict==1.13
easyocr==1.7.2
einops==0.8.0
einshape==1.0
et_xmlfile==2.0.0
executing==2.2.0
face-alignment==1.4.1
face-recognition-models==0.3.0
face-recognition==1.3.0
facexlib==0.3.0
fairscale==0.4.13
faiss-cpu==1.12.0
fal_client==0.7.0
fastapi==0.115.13
fastavro==1.12.0
faster-whisper==1.1.1
fastrlock==0.8.3
feedparser==6.0.11
ffmpeg-python==0.2.0
ffmpy==0.3.0
filelock==3.18.0
filetype==1.2.0
filterpy==1.4.5
fire==0.7.0
fish-audio-sdk==2025.6.3
flake8==7.3.0
flask-cors==6.0.1
flatbuffers==25.1.24
flet==0.26.0
fonttools==4.55.8
frozenlist==1.5.0
fsspec==2025.7.0
ftfy==6.3.1
future==1.0.0
fvcore==0.1.5.post20221221
gast==0.6.0
gdown==5.2.0
genson==1.3.0
gguf==0.17.1
gitdb==4.0.12
glfw==2.9.0
glitch_this==1.0.3
google-ai-generativelanguage==0.6.15
google-api-core==2.25.1
google-api-python-client==2.173.0
google-auth-httplib2==0.2.0
google-auth==2.40.3
google-cloud-core==2.4.3
google-cloud-storage==3.1.1
google-crc32c==1.7.1
google-genai==1.21.1
google-generativeai==0.8.5
google-pasta==0.2.0
google-resumable-media==2.7.2
google==3.0.0
googleapis-common-protos==1.70.0
googletrans-py==4.0.0
gradio==5.34.2
gradio_client==1.10.3
greenlet==3.2.3
griffe==1.12.1
groovy==0.1.2
groq==0.9.0
grpcio-status==1.62.3
grpcio-tools==1.62.3
grpcio==1.73.0
gunicorn==23.0.0
h11==0.14.0
h2==4.2.0
h5py==3.14.0
hf-xet==1.1.10
hpack==4.1.0
html2image==2.0.7
html5lib==1.1
httpcore==1.0.7
httplib2==0.22.0
httpx-sse==0.4.0
httpx-ws==0.7.2
httpx==0.28.1
huggingface-hub==0.34.4
humanfriendly==10.0
hydra-core==1.3.2
hyperframe==6.1.0
icecream==2.1.4
identify==2.6.12
idna==3.10
imageio-ffmpeg==0.6.0
imageio==2.37.0
imagesize==1.4.1
img2texture==1.0.6
importlib_metadata==8.6.1
inflate64==1.0.3
inflect==7.0.0
iniconfig==2.1.0
insightface==0.7.3
instructor==1.5.2
intel-pti==0.10.1
iopath==0.1.10
ipython==9.3.0
ipython_pygments_lexers==1.1.1
isodate==0.7.2
isort==6.0.1
itsdangerous==2.2.0
jaconv==0.4.0
jax==0.5.0
jaxlib==0.5.0
jedi==0.19.2
jieba==0.42.1
jiter==0.5.0
jmespath==1.0.1
joblib==1.4.2
json5==0.12.0
json_repair==0.47.1
jsonpatch==1.33
jsonpointer==3.0.0
jsonschema-specifications==2024.10.1
jsonschema==4.23.0
keras==3.10.0
keyboard==0.13.5
kiwisolver==1.4.8
kornia==0.8.0
kornia_rs==0.1.8
langchain-community==0.3.27
langchain-core==0.3.74
langchain-ollama==0.3.6
langchain-openai==0.3.30
langchain-text-splitters==0.3.9
langchain==0.3.27
langsmith==0.4.14
language-tags==1.2.0
lark-parser==0.12.0
lark==1.2.2
latex2mathml==3.78.0
lazy_loader==0.4
libclang==18.1.1
librosa==0.10.2
lightning-utilities==0.14.3
lightning==2.5.1.post0
lion-pytorch==0.2.3
litelama==0.1.7
llama-cloud-services==0.6.54
llama-cloud==0.1.35
llama-index-cli==0.5.0
llama-index-core==0.13.2
llama-index-embeddings-openai==0.5.0
llama-index-indices-managed-llama-cloud==0.9.2
llama-index-instrumentation==0.4.0
llama-index-llms-openai==0.5.4
llama-index-readers-file==0.5.2
llama-index-readers-llama-parse==0.5.0
llama-index-workflows==1.3.0
llama-index==0.13.2
llama-parse==0.6.54
llama_cpp_python==0.2.89
llvmlite==0.44.0
lmdb==1.6.2
loguru==0.7.3
loralib==0.1.2
lpips==0.1.4
lxml==5.3.0
lz4==4.4.4
manifold3d==3.0.1
mapbox_earcut==1.0.3
markdown-it-py==3.0.0
markdownify==1.2.0
marshmallow==3.26.1
matplotlib-inline==0.1.7
matplotlib==3.10.0
matrix-client==0.4.0
matrix-nio==0.25.2
mccabe==0.7.0
mcp==1.13.0
mdtex2html==1.3.1
mdurl==0.1.2
mediapipe==0.10.9
mkl==2025.1.0
ml-dtypes==0.4.1
modelscope==1.27.0
more-itertools==10.7.0
moviepy==1.0.3
mpmath==1.3.0
msgpack==1.1.1
mss==10.0.0
mtcnn==1.0.0
much==0.0.7
multidict==6.1.0
multivolumefile==0.2.3
mypy_extensions==1.1.0
namex==0.1.0
narwhals==1.43.1
natsort==8.4.0
neo4j==5.28.2
nest-asyncio==1.6.0
networkx==3.5
ninja==1.11.1.4
nltk==3.9.1
nodeenv==1.9.1
numba==0.61.0
numexpr==2.11.0
numpy==1.26.4
nunchaku==1.0.1.dev20250926+torch2.7
nvidia-ml-py==12.570.86
oauthlib==3.2.2
ollama==0.5.1
omegaconf==2.3.0
onnx2torch==1.5.15
onnx==1.17.0
onnxruntime-gpu==1.22.0
onnxruntime==1.20.1
open_clip_torch==2.26.1
openai-whisper==20240930
openai==1.99.9
opencv-contrib-python-headless==4.11.0.86
opencv-contrib-python==4.11.0.86
opencv-python-headless==4.11.0.86
opencv-python==4.12.0.88
openpyxl==3.1.5
opt_einsum==3.4.0
optimum==1.27.0
optree==0.16.0
orderly-set==5.3.0
orjson==3.10.18
ormsgpack==1.10.0
outcome==1.3.0.post0
packaging==24.2
pandas==2.2.3
parso==0.8.4
pathlib==1.0.1
pathspec==0.12.1
pdf2image==1.17.0
pdfminer.six==20250506
pdfplumber==0.11.7
peft==0.17.0
phonemizer==3.3.0
piexif==1.1.3
pilgram==1.2.1
pillow==11.3.0
pip==24.0
pixeloe==0.1.1
platformdirs==4.3.8
pluggy==1.6.0
polars==1.32.3
pooch==1.8.2
portalocker==3.1.1
pre_commit==4.2.0
prettytable==3.16.0
proces==0.1.7
prodigy-plus-schedule-free==1.9.2
prodigyopt==1.1.2
proglog==0.1.12
prompt_toolkit==3.0.51
propcache==0.2.1
proto-plus==1.26.1
protobuf==3.20.3
psd-tools==1.10.8
psutil==6.1.1
pure_eval==0.2.3
py-cord==2.6.1
py-cpuinfo==9.0.0
py3langid==0.2.2
py7zr==0.22.0
py==1.11.0
pyOpenSSL==25.1.0
pyarrow==20.0.0
pyasn1==0.6.1
pyasn1_modules==0.4.2
pybcj==1.0.6
pyclipper==1.3.0.post6
pycocoevalcap==1.2
pycocotools==2.0.10
pycodestyle==2.14.0
pycollada==0.8
pycparser==2.22
pycryptodome==3.23.0
pycryptodomex==3.23.0
pydantic-settings==2.9.1
pydantic==2.11.7
pydantic_core==2.33.2
pydeck==0.9.1
pydub==0.25.1
pyflakes==3.4.0
pykakasi==2.3.0
pykalman==0.10.1
pyloudnorm==0.1.1
pynvml==12.0.0
pyopenjtalk==0.4.1
pyparsing==3.2.1
pypdf==6.0.0
pypdfium2==4.30.0
pypinyin==0.54.0
pyppmd==1.1.1
pyreadline3==3.5.4
pytest==8.4.1
python-bidi==0.6.6
python-dateutil==2.9.0.post0
python-dotenv==1.1.0
python-engineio==4.12.2
python-multipart==0.0.20
python-socketio==5.13.0
python-socks==2.7.1
pytorch-extension==0.2
pytorch-lightning==2.5.5
pytz==2025.1
pyvips==2.1.0
pywin32==311
pyzbar==0.1.9
pyzstd==0.17.0
qrcode==8.2
qwen-vl-utils==0.0.11
rdflib==7.1.4
redis==6.2.0
referencing==0.36.2
regex==2024.11.6
rembg==2.0.62
repath==0.9.0
replicate==1.0.7
reportlab==4.3.0
requests-toolbelt==1.0.0
requests==2.32.3
requirements-parser==0.13.0
retina-face==0.0.17
retry==0.9.2
rfc3986==1.5.0
rich-argparse==1.7.1
rich==13.9.4
rotary-embedding-torch==0.8.9
rpds-py==0.22.3
rsa==4.9.1
ruamel.yaml.clib==0.2.12
ruamel.yaml==0.18.14
ruff==0.12.0
runwayml==3.5.0
s3transfer==0.13.0
sacremoses==0.0.53
safehttpx==0.1.6
safetensors==0.5.2
sageattention==2.2.0+cu128torch2.7.1.post2
scenedetect==0.6.6
schedulefree==1.4.1
scikit-base==0.12.3
scikit-image==0.25.2
scikit-learn==1.6.1
scipy==1.10.1
seaborn==0.13.2
segment-anything==1.0
segments==2.3.0
selenium==4.35.0
semantic-version==2.10.0
sentence-transformers==5.1.0
sentencepiece==0.2.0
setuptools==80.9.0
sgmllib3k==1.0.0
shapely==2.0.7
shellingham==1.5.4
shtab==1.7.2
simple-websocket==1.1.0
simpleeval==1.0.3
simsimd==6.2.1
six==1.17.0
smmap==5.0.2
sniffio==1.3.1
sortedcontainers==2.4.0
sounddevice==0.5.1
soundfile==0.13.1
soupsieve==2.6
soxr==0.5.0.post1
spandrel==0.4.1
srt==3.5.3
sse-starlette==3.0.2
stack-data==0.6.3
stanza==1.1.1
starlette==0.46.2
streamlit==1.46.0
stringzilla==3.11.3
striprtf==0.0.26
surrealist==1.1.2
svg.path==6.3
svglib==1.5.1
swarm==0.1.0
sympy==1.14.0
tabulate==0.9.0
tbb==2022.1.0
tenacity==8.5.0
tensorboard-data-server==0.7.2
tensorboard==2.17.1
tensorflow-intel==2.17.1
tensorflow-io-gcs-filesystem==0.31.0
tensorflow==2.17.1
termcolor==2.5.0
texttable==1.7.0
tf_keras==2.17.0
threadpoolctl==3.5.0
tifffile==2025.1.10
tiktoken==0.9.0
timm==1.0.15
tinycss2==1.4.0
tokenizers==0.21.4
tomesd==0.1.3
toml==0.10.2
tomli==2.2.1
tomlkit==0.13.3
toolz==1.0.0
torch==2.7.1+cu128
torchaudio==2.7.1+cu128
torchmetrics==1.7.3
torchsde==0.2.6
torchtyping==0.1.5
torchvision==0.22.1+cu128
tornado==6.5.1
tqdm==4.67.1
traitlets==5.14.3
trampoline==0.1.2
transformers-stream-generator==0.0.5
transformers==4.53.3
transparent-background==1.3.2
trimesh==4.6.1
trio-websocket==0.12.2
trio==0.30.0
triton-windows==3.1.0.post17
triton==3.1.0
typeguard==2.13.3
typer-config==1.4.2
typer==0.15.1
types-requests==2.32.4.20250809
typing-inspect==0.9.0
typing-inspection==0.4.1
typing_extensions==4.14.1
tyro==0.8.5
tzdata==2025.1
ultralytics-thop==2.0.14
ultralytics==8.3.169
unpaddedbase64==2.1.0
uritemplate==4.2.0
urllib3==2.5.0
uv==0.5.27
uvicorn==0.34.3
verovio==5.5.0
vhacdx==0.0.8.post1
virtualenv==20.31.2
volcengine==1.0.200
voluptuous==0.15.2
watchdog==6.0.0
wcwidth==0.2.13
webcolors==24.11.1
webencodings==0.5.1
webrtcvad-wheels==2.0.14
webrtcvad==2.0.10
websocket-client==1.8.0
websockets==15.0.1
wget==3.2
wheel==0.45.1
wikipedia==1.4.0
win32_setctime==1.2.0
wrapt==1.17.2
wsproto==1.2.0
xatlas==0.0.9
xformers==0.0.31.post1
xlrd==2.0.2
xxhash==3.5.0
yacs==0.1.8
yapf==0.43.0
yarl==1.18.3
zhipuai==2.1.5.20250611
zipp==3.21.0
zstandard==0.24.0

**Comment by xlibfly** [comment]
我升级了torch和cuda后来跑通了
